test

```diff
@@ -28,16 +28,35 @@ fi
 echo "Checking govet..."
 go vet $PKG_VET
 
+# This is needed to prevent the race detector from crashing, but only when run from
+# within the SDK, and only sometimes. It appears the issue lies with cgo (since adding
+# this fixes it) and the go race detection (which uses libsanitize). It looks like the
+# kernel could be involved too, since thats the case in this similar but different bug:
+# https://github.com/google/sanitizers/issues/837
+# I tried upgrading to a 4.13 kernel as suggested by that link, but it didn't help.
+# Race detector documentation is sparse, as is cgo documentation, and documentation on
+# race detection with cgo is practically nonexistant.
+# The weirdest part is that it fails by trying to call mmap with an address thats 49 bits
+# big, which is notably not a valid x86_64 virtual address.
+# Locksmith has a similar fix:
+# https://github.com/coreos/coreos-overlay/commit/3727e78422975bdad631ded0d240da51d1c684d5
+# TODO(ajeddeloh): find out why this is needed.
```

Comment:
FWIW I find that `TODO`s often don't get followed up on. Maybe delete this and make a github issue?

Reply:
It's not really a bug yet, since I can't trigger it until I run this. I can file a bug in advance with the workaround in place or file it once it merges. I don't really see the harm in the TODO, especially with a name; at the very least it shows who might know something about the issue (granted that can be found with `git blame`).

Reply:
There's no harm, just a thought I had. Feel free to ignore.